package.json

```diff
@@ -2,10 +2,13 @@
   "name": "truffle-cli",
   "version": "0.0.8",
   "description": "",
-  "main": "index.js",
+  "main": "dist/cli.js",
   "type": "module",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "postinstall": "npm run build",
+    "build": "tsc && chmod +x dist/cli.js",
```

Comment:
When I run `yarn build` it seems to be putting all the files in `dist/src/`, so the chmod command fails. It will probably also fail to run if anyone installs it :sweat_smile:. I forget, but is there a way to move all of the files up one directory?